test/addons/openssl-binding/binding.gyp

```diff
@@ -2,8 +2,12 @@
   'targets': [
     {
       'target_name': 'binding',
-      'sources': ['binding.cc'],
-      'include_dirs': ['../../../deps/openssl/openssl/include'],
+      'conditions': [
+         ['node_use_openssl=="true"', {
+           'sources': ['binding.cc'],
+           'include_dirs': ['../../../deps/openssl/openssl/include'],
+         }]
+      ]
```

Comment:
I'm kind of surprised this works.  gyp generally doesn't like it when there are no source files to compile.